addons/block_code/simple_nodes/simple_scoring/simple_scoring.gd

```diff
@@ -1,48 +1,32 @@
 @tool
 class_name SimpleScoring
-extends CanvasLayer
+extends Node2D
 
 const BlockDefinition = preload("res://addons/block_code/code_generation/block_definition.gd")
 const BlocksCatalog = preload("res://addons/block_code/code_generation/blocks_catalog.gd")
 const Types = preload("res://addons/block_code/types/types.gd")
 
-@export var score_left: int:
-	set = _set_score_left
+@export var score: int:
+	set = _set_score
 
-@export var score_right: int:
-	set = _set_score_right
+var _score_label: Label
 
-var _score_labels: Dictionary
 
-const _POSITIONS_FOR_PLAYER = {
-	"1": "left",
-	"2": "right",
-}
+func _create_score_label():
+	if _score_label:
+		return
 
+	_score_label = Label.new()
```

Comment:
Why not just make this extend Label instead of having a child label?

Reply:
I played with that :) My reason here is I felt it's a _feature_ that SimpleScoring provides a simple interface in the editor: no label properties, no extra Control-related handles in the canvas. But with that in mind, I created #249 to add a block for setting a label's text.